packages/util/src/RawFile.ts

```diff
@@ -0,0 +1,31 @@
+import type { Buffer } from 'node:buffer';
+
+/**
+ * Represents a file to be added to a request with multipart/form-data encoding
+ */
+export interface RawFile {
+	/**
+	 * Content-Type of the file (e.g., 'image/png', 'application/pdf')
+	 * If not provided, it will be inferred from the file data when possible
```

Comment:
```suggestion
	 * Content-Type of the file.
	 * If not provided, it will be inferred from the file data when possible
	 *
	 * @example 'image/png'
	 * @example 'application/pdf'
```